SncRedisBundle.php

```diff
@@ -23,6 +24,16 @@
  */
 class SncRedisBundle extends Bundle
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function boot()
+    {
+        if (!class_exists(Client::class) && file_exists($file = $this->container->getParameter('kernel.cache_dir').'/snc_phpredis_client.php')) {
```

Comment:
What do you think about throwing an exception if a phpredis client is defined but the file is not present in the cache ?

The mechanism would be more robust and we could identify mistakes easily.